From 65f981662754c41bde25f87458044205c6db9a29 Mon Sep 17 00:00:00 2001
From: Antony Pavlov <antonynpavlov@gmail.com>
Date: Tue, 21 Feb 2012 02:17:35 +0400
Subject: [PATCH] sched.h dirty hack

---
 src/SDL/i_system.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/SDL/i_system.c b/src/SDL/i_system.c
index 8a60258..93622ad 100644
--- a/src/SDL/i_system.c
+++ b/src/SDL/i_system.c
@@ -65,6 +65,8 @@
 #endif
 #ifdef HAVE_SCHED_H
 #include <sched.h>
+#define CPU_SET(cpu, cpusetp)	__CPU_SET (cpu, cpusetp)
+#define CPU_ZERO(cpusetp)	__CPU_ZERO (cpusetp)
 #endif
 #ifdef _MSC_VER
 #include <io.h>
-- 
1.7.8.3

